automation/src/test/java/com/microsoft/identity/common/test/automation/AquireTokenUnknownAuthorityAuthorityValidationFalse.java

```diff
@@ -0,0 +1,116 @@
+import com.microsoft.identity.common.test.automation.tasks.ReadCache;
+import com.microsoft.identity.common.test.automation.utility.Scenario;
+import com.microsoft.identity.common.test.automation.utility.TestConfigurationQuery;
+
+import net.serenitybdd.junit.runners.SerenityParameterizedRunner;
+import net.serenitybdd.screenplay.abilities.BrowseTheWeb;
+import net.thucydides.core.annotations.Managed;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.junit.annotations.TestData;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+
+import io.appium.java_client.service.local.AppiumDriverLocalService;
+
+import static net.serenitybdd.screenplay.GivenWhenThen.seeThat;
+import static net.serenitybdd.screenplay.GivenWhenThen.then;
+import static org.hamcrest.Matchers.is;
+
+@RunWith(SerenityParameterizedRunner.class)
+public class AquireTokenUnknownAuthorityAuthorityValidationFalse {
+
+    @TestData
+    public static Collection<Object[]> FederationProviders(){
+
+        return Arrays.asList(new Object[][]{
+                {"ADFSv2"},
+                {"ADFSv3"},
+                {"ADFSv4"}
+        });
+
+    }
+
+    static AppiumDriverLocalService appiumService = null;
+
+    @Managed(driver="Appium")
+    WebDriver hisMobileDevice;
+
+    @BeforeClass
+    public static void startAppiumServer() throws IOException {
+        appiumService = AppiumDriverLocalService.buildDefaultService();
+        appiumService.start();
+    }
+
+    @AfterClass
+    public static void stopAppiumServer() {
+        appiumService.stop();
+    }
+
+    private User james;
+    private String federationProvider;
+
+    @Steps
+    AcquireToken acquireToken;
+
+    @Steps
+    ReadCache readCache;
+
+    @Steps
+    ClickDone clickDone;
+
+    public AquireTokenUnknownAuthorityAuthorityValidationFalse(String federationProvider){
+        this.federationProvider = federationProvider;
+    }
+
+    @Before
+    public void jamesCanUseAMobileDevice(){
+        TestConfigurationQuery query = new TestConfigurationQuery();
+        query.federationProvider = this.federationProvider;
+        query.isFederated = true;
+        query.userType = "Member";
+        james = getUser(query);
+        james.can(BrowseTheWeb.with(hisMobileDevice));
+    }
+
+    private static User getUser(TestConfigurationQuery query){
+
+        Scenario scenario = Scenario.GetScenario(query);
+
+        User newUser = User.named("james");
+        newUser.setFederationProvider(scenario.getTestConfiguration().getUsers().getFederationProvider());
+        newUser.setTokenRequest(scenario.getTokenRequest());
+        newUser.setCredential(scenario.getCredential());
+        newUser.setAuthority("https://myfakeauthority.com");
```

Comment:
This throws an IllegalArgumentException as the authority validation looks for two slashes after https:// in the url. I assume this is by design and intended. Please confirm this with the team and change the fake url to something like this : https://myfakeauthority.com/fake/